service/src/main/java/uni/miskolc/ips/ilona/tracking/service/gateway/PositionQueryServiceSIConfig.java

```diff
@@ -91,23 +95,18 @@ public HttpRequestExecutingMessageHandler httpGateway() {
         gateway.setHttpMethod(HttpMethod.GET);
         gateway.setExpectedResponseType(String.class);
         gateway.setOutputChannel(setAlgorithmReplyChannel());
+        LOG.info("algorithm set in Positioning server using Spring Integration");
         return gateway;
     }
 
     @Bean
     @ServiceActivator(inputChannel = "getLocationQueryChannel")
     public HttpRequestExecutingMessageHandler locationGateway() {
-        //System.out.println("fsaz");
```

Comment:
I don't know why this command had been committed in the first place.

Reply:
oops